File: mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    // rejects answers less than 1
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);

    // r for rows, l for row #, b for # of blocks/hashes for row
    for (int r = 1; r <= n; r++)

    {
        for (int s = 1; s <= (n - r); s++)
        {
            printf(" ");
        }
        for (int b = 1; b <= r; b++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break
    space = height
    block = 1
for i in range(height):
    print(" " * space + "#" * block + "  " + "#" * block)
    space -= 1
    block += 1
